libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/ActivityUtils.java

```diff
@@ -13,4 +15,11 @@ public static void hideKeyboard(Activity activity) {
                     InputMethodManager.HIDE_NOT_ALWAYS);
         }
     }
+
```

Comment:
This would benefit from a comment explaining the difference between `hideKeyboard` and `hideKeyboardForced`.

Reply:
👍 added in 95cabf2.